vizro-core/src/vizro/actions/_callback_mapping/_callback_mapping_utils.py

```diff
@@ -137,7 +154,7 @@ def _get_action_callback_inputs(action_id: ModelID) -> Dict[str, List[State]]:
         "filter_interaction": (
             _get_inputs_of_chart_interactions(action_id=action_id, action_function=filter_interaction.__wrapped__)
             if "filter_interaction" in include_inputs
-            else []
+            else {}
```

Comment:
I probably need to adjust all inputs formats to the dictionary format filter_interaction use.